pod/annotations.go

```diff
@@ -236,10 +236,10 @@ const (
 	AnnotationKeyPodParameterMockPodKillTime    = "mockPod.netflix.com/killTime"
 
 	// version recording; this is output from titus-executor mostly used
-	// for debugging. if we decide to add more versions here, let's make it
-	// one annotation with csv or similar.
-	AnnotationKeyRuntimeVersionTitusExecutor = "runtime.version.titus.netflix.com/titus-executor"
-	AnnotationKeyRuntimeVersionLinuxKernel   = "runtime.version.titus.netflix.com/linux-kernel"
+	// for debugging.
+	AnnotationKeyRuntimeVersions                = "runtime.titus.netflix.com/versions"
+	AnnotationMemberRuntimeVersionTitusExecutor = "titus-executor"
+	AnnotationMemberRuntimeVersionLinuxKernel   = "linux-kernel"
 )
```

Comment:
I'm telling you, this is overkill.

Just have one annotation `AnnotationKeyRuntimeVersions` and let it be a free-form CSV until you know you need something more structured

Reply:
Sure, I can drop these.